code/modules/mining/abandoned_crates.dm

```diff
@@ -213,6 +214,12 @@
 			return
 	return ..()
 
+/obj/structure/closet/secure/loot/dive_into(mob/living/user)
+	if(!locked)
+		return ..()
+	to_chat(user, "<span class='notice'>That seems like a stupid idea.</span>")
```

Comment:
Literally idiot proofing more shit. the fuck.

Reply:
It's more of an unexpected wacky interaction between features. dive_into() will try to togglelock() the loot crate, only for it to explode because that's its behaviour.
You should keep in mind that loot crates code is pretty old anyway.